project/fr-FR/Mod/ExtendedRadio.json

```diff
@@ -11,5 +11,5 @@
   "Options.OPTION_DESCRIPTION[ExtendedRadio.ExtendedRadio.Mod.Setting.ReloadRadio]": "Recharger les radios, peut mettre à jour votre radio si vous l'avez modifiée. Peut aussi faire planter le jeu.",
   "Options.OPTION[ExtendedRadio.ExtendedRadio.Mod.Setting.ResetSettings]": "Réinitialiser les paramètres d'ExtendedRadio",
   "Options.OPTION_DESCRIPTION[ExtendedRadio.ExtendedRadio.Mod.Setting.ResetSettings]": "Réinitialise les paramètres du mod à leur valeur par défaut.",
-  "Options.WARNING[ExtendedRadio.ExtendedRadio.Mod.Setting.ResetSettings]": "Êtes-vous sûr de vouloir réinitialiser les paramètres d'ExtendedRadio?"
-}
\ No newline at end of file
+  "Options.WARNING[ExtendedRadio.ExtendedRadio.Mod.Setting.ResetSettings]": "Êtes-vous sûr de vouloir réinitialiser les paramètres d'ExtendedRadio ?"
```

Comment:
why space

Reply:
@baka-gourd French requires space before "?", "!" and ";".